src/utils/transition/linear-interpolator.js

```diff
@@ -42,6 +61,18 @@ export default class LinearInterpolator extends TransitionInterpolator {
     for (const key of this.propNames) {
       viewport[key] = lerp(startProps[key], endProps[key], t);
     }
+
+    if (endProps.around) {
+      // zoom around provided point
+      const [longitude, latitude] = new WebMercatorViewport(Object.assign({}, endProps, viewport))
+        .getMapCenterByLngLatPosition({
```

Comment:
Can we apply this logic and calculate lng/lat during  https://github.com/uber/react-map-gl/blob/837588c9272e254f7c86941d02ca2705f058c6a4/src/utils/map-controls.js#L364

Reply:
This might guarantee the screen/lng-lat values match only at ends, but not sure if the arc style transition gets fixed or not.

Reply:
The start/end of the transition was already calculated correctly. The problem is that the transition of longitude/latitude is not linear during the transition. When we treated them as linear we get the weird arc.

Reply:
I see, thanks for the details, looks good to me, already approved.